### PR TITLE
[M] update mainpage to baisc canlender with FullCalendar lib

### PR DESCRIPTION
**Pull request by penlin**
[M] update mainpage to baisc canlender with FullCalendar lib

### CODE REVIEW

File: web/main.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 import os
 import ws_api
-from bottle import route, run, template
-
+from bottle import route, run, template, get
 
+@get('/')
+def test():
+	return template('tpl/mentholatum')
```
Comment (bruce3557) [inline web/main.py]: 2 space indent

File: web/tpl/mentholatum.tpl [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+	<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/fullcalendar/2.4.0/fullcalendar.min.css' />
```
Comment (bruce3557) [inline web/tpl/mentholatum.tpl]: Ditto

**Comment by penlin** [comment]
fix indent problem with 2 spaces

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**